packages/ui-scripts/lib/publish.js

```diff
@@ -30,19 +30,34 @@ const { getConfig } = require('./utils/config')
 
 try {
   const pkgJSON = getPackageJSON()
-
+  // Arguments
+  // 1: publish type. defaults to latest. If set to 'maintenance', it will publish with vx_maintenance tag
+  // e.g.: ui-scripts --publish maintenance
   publish({
     packageName: pkgJSON.name,
     currentVersion: pkgJSON.version,
-    packageConfig: getConfig(pkgJSON)
+    packageConfig: getConfig(pkgJSON),
+    releaseType: process.argv[3]
```

Comment:
The `fixPublish` function should have the same parameter signature as this function.